tests/repeated-updates.sh

```diff
@@ -0,0 +1,39 @@
+#! /bin/sh -e
+
+SCRATCH=scratch/$(basename $0 .sh)
+PATCHELF=$(readlink -f "../src/patchelf")
+
+rm -rf ${SCRATCH}
```

Comment:
Can you run shellcheck on this file? https://p.krebsco.de/1fq0m92

Reply:
I will add this to our CI later.

Reply:
Sure. I'll do that later on tonight.